Automattic/jetpack
# Admin Page: refactor core external link component

Fixes #22520

#### Changes proposed in this Pull Request:

* Replace `ExternalLink` component to use `core` package import.

#### Jetpack product discussion NO



#### Does this pull request change what data or activity we track or use? NO



#### Testing instructions:






* build this PR
* Check if all external links works properly same as previously
* all test cases are passing
